scripts/ci/update-github

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -ex
+
+TAG_NAME=$(git tag -l --contains HEAD)
+
+rm -rf dcos-nightly
+
+# Setup Git
+git config --global user.email "$GIT_USER@users.noreply.github.com"
+git config --global user.name "MesosphereCI Robot"
+git config credential.helper 'cache --timeout=300'
+git clone "https://$GIT_USER:$GIT_PASSWORD@github.com/mesosphere/dcos" dcos-nightly
+cd dcos-nightly
+
+if [[ $TAG_NAME ]]
+then
+  git checkout -- .
+  git remote add upstream "https://$GIT_USER:$GIT_PASSWORD@github.com/dcos/dcos"
+  git checkout master
+  git checkout -b dcos-ui/$TAG_NAME
+
+  cp -fR ../buildinfo.json packages/dcos-ui/buildinfo.json
+
+  git add "packages/dcos-ui/buildinfo.json"
+  git commit -m "chore(dcos-ui): update package"
+  git push --force origin dcos-ui/$TAG_NAME
+
+  cd ../
+
+  CHANGELOG=$(TAG_NAME=$TAG_NAME ./scripts/ci/generate-changelog)
```

Comment:
I think the generate-changelog script is not dependent on the folder it is called from, at least as far as I could tell, so let's spare the `cd ../` and `cd dcos-nightly` and make it `../scripts/ci/generate-changelog`. The reason why I'm mentioning this is that I often find it quite hard to keep track of where I am directory wise in long bash scripts. Removing these parts free my mind a lot.

Reply:
Ah yes. We must do it this way, because we need to be in the proper `.git` context. The dcos-nightly has it's own git context, and we don't want the changelog to use this context.